validator/views/validation.py

```diff
@@ -207,12 +207,48 @@ def stop_validation(request, result_uuid):
 @login_required(login_url='/login/')
 def validation(request):
     dc_prefix = 'datasets'
-    ref_repfix = 'ref'
-    data_initial_values = [{'filters': DataFilter.objects.filter(name='FIL_ALL_VALID_RANGE'),
-                            'dataset': Dataset.objects.get(short_name=val_globals.C3S), }]
-
-    ref_initial_values = {'filters': DataFilter.objects.filter(name='FIL_ALL_VALID_RANGE'),
-                          'dataset': Dataset.objects.get(short_name=val_globals.ISMN), }
+    ref_prefix = 'ref'
+
+    # initial values
+    valrun_uuid = request.GET.get("valrun_uuid", None)
+    if valrun_uuid == None:
+        ref_initial_values = {
+            "filters": DataFilter.objects.filter(name="FIL_ALL_VALID_RANGE"),
+            "dataset": Dataset.objects.get(short_name=val_globals.ISMN),
+        }
+        data_initial_values = [{
+            "filters": DataFilter.objects.filter(name="FIL_ALL_VALID_RANGE"),
+            "dataset": Dataset.objects.get(short_name=val_globals.C3S),
+        }]
+        valrun_initial_values = None
+    else:
+        valrun = get_object_or_404(ValidationRun, pk=valrun_uuid)
+        # initial settings for datasets
+        ref_config = valrun.reference_configuration
+        data_configs = [dc for dc in valrun.dataset_configurations.all()
+                        if dc.id != ref_config.id]
+        initial_values = [
+            {
+                "filters": dc.filters.all(),
+                "dataset": dc.dataset,
+                "version": dc.version,
+                "variable": dc.variable,
+            }
+            for dc in [ref_config] + data_configs
+        ]
+        ref_initial_values = initial_values[0]
+        data_initial_values = initial_values[1:]
```

Comment:
This is how I infer the initial filters to set.

Reply:
I think I could also infer the initial filters from `request.POST` for a POST request